Source/SizableCollectionCollectionDataDisplayManager/Support/SizableCollectionCellGenerator.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  HeightCollectionCellGenerator.swift
+//  ReactiveDataDisplayManager
+//
+//  Created by Alexander Filimonov on 02/03/2020.
+//  Copyright © 2020 Александр Кравченков. All rights reserved.
+//
+
+import Foundation
+
+/// Generator that describes collection cell generator that can returns size
+public protocol SizableCollectionCellGenerator: class {
```

Comment:
Should we moved this file into the protocols to the another protocols?)

Reply:
maybe) I will do this